data_steward/resource_files/schemas/rdr/questionnaire_response_additional_info.json

```diff
@@ -0,0 +1,20 @@
+[
+    {
+        "type": "integer",
+        "name": "questionnaire_response_id",
+        "mode": "nullable",
+        "description": ""
```

Comment:
This should be `"mode": "required",` and given a fitting description.
"The value used to map a participant's responses to a survey submission."

The rest of this file exists in our code base and should not need to be updated.  You should rebase this branch to get the correct version of this file.